README.rst

```diff
@@ -188,11 +188,75 @@ Using CWL:
 
     $ cwltool --outdir=./outputs ./workflow/workflow.cwl ./workflow/input.yaml
 
+Create REANA file
+=================
```

Comment:
We now use a different wording with the release of REANA v0.3.0, please see the example https://github.com/reanahub/reana-demo-worldpopulation/  Could you amend the wording here as well?